src/IJ_mv/IJVector_parcsr.c

```diff
@@ -264,6 +268,42 @@ hypre_IJVectorZeroValuesPar(hypre_IJVector *vector)
    return hypre_error_flag;
 }
 
+/******************************************************************************
+ *
+ * hypre_IJVectorSetComponentPar
+ *
+ * Set the component identifier of a vector with multiple components
+ * (multivector)
+ *
+ *****************************************************************************/
+
+HYPRE_Int
+hypre_IJVectorSetComponentPar(hypre_IJVector *vector,
+                              HYPRE_Int       component)
+{
+   HYPRE_Int        print_level = hypre_IJVectorPrintLevel(vector);
+   hypre_ParVector *par_vector  = (hypre_ParVector*) hypre_IJVectorObject(vector);
+   HYPRE_Int        num_vectors = hypre_ParVectorNumVectors(par_vector);
+
+   if (component < 0 || component > num_vectors)
+   {
+      if (print_level)
+      {
+         hypre_printf("component < 0 || component > num_vectors -- ");
+         hypre_printf("hypre_IJVectorSetComponentPar\n");
+         hypre_printf("**** This vector partitioning should not occur ****\n");
```

Comment:
this message about 'partitioning' doesn't seem to be relevant

Reply:
Removed. Thanks!